docs/cookbook/deployment/platform-sh.rst

```diff
@@ -203,6 +203,62 @@ When you get connected please run:
     By default platform.sh creates only one instance of the database with the ``main`` name.
     Platform.sh works with the concept of an environment per branch if activated. The idea is to mimic production settings per each branch.
 
+.. rst-class:: plus-doc
+
+Sylius Plus Deployment
+----------------------
+
+`Sylius Plus <https://sylius.com/plus/>`_ is installed to Sylius like a plugin, but it needs some changes in configuration to deploy it properly.
+First of all you need configure your project follow by `Sylius Plus installation </book/installation/sylius_plus_installation>`_
+After that you can modify your the ``.platform.app.yaml``
```

Comment:
```suggestion
After that, you should modify your the ``.platform.app.yaml``
```

Reply:
I'd mention which step numbers you need to alternate from the original Platform.sh deployment guide you have above.